PhysicsTools/PatUtils/python/tools/muonRecoMitigation.py

```diff
@@ -0,0 +1,84 @@
+import FWCore.ParameterSet.Config as cms
+
+from FWCore.GuiBrowsers.ConfigToolBase import *
+
+def muonRecoMitigation(process,
+                       pfCandCollection,
+                       runOnMiniAOD,
+                       selection="",
+                       muonCollection="",
+                       cleanCollName="cleanMuonsPFCandidates",
+                       cleaningScheme="all",
+                       postfix=""):
+
+    sequence=cms.Sequence()    
+
+    if selection=="":
+        if runOnMiniAOD:
+            from RecoMET.METFilters.badGlobalMuonTaggersMiniAOD_cff import badGlobalMuonTagger, cloneGlobalMuonTagger
+        else:
+            from RecoMET.METFilters.badGlobalMuonTaggersAOD_cff import badGlobalMuonTagger, cloneGlobalMuonTagger
```

Comment:
the way these are loaded, better define different module names in the _cff files and then load appropriately named clones.
Else, there is a chance a wrong load elsewhere overrides the result